app/util/data_preparation/jira_prepare_data.py

```diff
@@ -153,6 +153,14 @@ def __check_current_language(jira_api):
                          f'Please change your profile language to "English (United States) [Default]"')
 
 
+def __check_for_admin_permissions(jira_api):
+    user_permissions = jira_api.get_user_permissions()
+    if not any((user_permissions['permissions']['ADMINISTER']['havePermission'],
+                user_permissions['permissions']['SYSTEM_ADMIN']['havePermission'])):
```

Comment:
Could you pls check if `user_permissions['permissions']['ADMINISTER_PROJECTS']` if sufficient permissions level for toolkit needs.

Reply:
unfortunately no